frontend/components/ProfilePublication/index.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react'
+
+import Publication from './Publication'
+
+const datas = [
+  { title: 'Publication1', desc: 'Description1!', link: 'google.com' },
+
+  { title: 'Publication2', desc: 'Description2!', link: 'google.com' }
+]
+
+class ProfilePublication extends Component {
+  render() {
+    return (
+      <>
+        {style}
+        <div className="grey lighten-1 profileContent">
+          <div className="col s12" style={{ padding: 0 }}>
+            <h4 style={{ margin: '0', color: '#0f6489' }}>Publications</h4>
+            <div>
+              {datas.map(data => (
+                <Publication data={data} />
+              ))}
```

Comment:
Hey mila, it is good practice to try using a `key` in React. I can explain to you why in the next session, but I am pretty sure you learned this in freecodecamp.

What I want to point out here is that sometimes it is good to surround your items in div when the map is trying to return two elements (publication returns a `a` and a `p` tag)

Take a look at this
![image](https://user-images.githubusercontent.com/44391389/75094054-641f2300-55c2-11ea-9126-85860c362166.png)

If you surround it with a div tag with some id or a className, you can easily group together related items. Just some tips, but nothing code breaking